packages/rx_bloc_test/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [4.1.0] - December 1, 2023
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package